examples/provider/provider-in-cluster.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: pkg.crossplane.io/v1
```

Comment:
Would it make more sense/be easier to read if the resources are listed in the order that they need to be created?  I know reconciliation will resolve ordering issues, but it might be more readable if it starts with `ServiceAccount` then `ClusterRoleBinding` then `DeploymentRuntimeConfig` and then `Provider`.

Reply:
Split the File only for ordered creation - can add it in one file with an comment

Reply:
We don't need to create the ServiceAccount - RuntimeConfig will create the ServiceAccount for you